ui/components/app/modals/contract-details-modal/index.scss

```diff
@@ -0,0 +1,99 @@
+.contract-details-modal {
+  background-color: var(--color-background-default);
+  width: 360px !important;
+
+  &__content {
+    padding: 24px 16px 32px 16px;
+    border-bottom: 1px solid var(--color-border-muted);
+
+    &__header {
+      font-style: normal;
+      font-weight: 700;
+      font-size: 16px;
+      line-height: 24px;
+      display: flex;
+      align-items: center;
+    }
+
+    &__description {
+      font-style: normal;
+      font-weight: 400;
+      font-size: 12px;
+      line-height: 20px;
+      display: flex;
+      align-items: center;
+      letter-spacing: 0.25px;
+      padding-top: 8px;
+      color: var(--color-text-alternative);
+    }
+
+    &__contract-subtitle {
+      font-style: normal;
+      font-weight: 400;
+      font-size: 14px;
+      line-height: 20px;
+      display: flex;
+      align-items: center;
+      padding-top: 16px;
+      padding-bottom: 8px;
+    }
+
+    &__contract {
+      border: 1px solid var(--color-border-default);
+      border-radius: 4px;
+      display: flex;
+      width: 100%;
+      height: 78px;
+
+      &__identicon {
+        margin: 16px 16px 38px 16px;
+      }
+
+      &__title {
+        font-weight: 700;
+        font-size: 16px;
+        margin-top: 16px;
+        width: 176px;
+        height: 24px;
+      }
+
+      &__address {
+        color: var(--color-text-alternative);
+        font-style: normal;
+        font-weight: 400;
+        font-size: 14px;
+        width: 176px;
+        height: 22px;
+        line-height: 22px;
+        margin-bottom: 16px;
+      }
+
+      &__buttons {
+        margin: 18px 20px 40px 1px;
+        display: flex;
+        justify-content: space-between;
+        width: 100%;
+        background-color: var(--color-background-default);
+      }
+
+      &__button {
+        background-color: var(--color-background-default);
+      }
+
+      &__buttonCopy {
+        background-color: var(--color-background-default);
+      }
+    }
+  }
+
+  &__footer {
+    @extend %row-nowrap;
```

Comment:
Don't extend anything defined in the ui/css/itcss folder. That is a deprecated folder that I honestly thought I had killed long ago. Use a Box, flex properties etc.

Reply:
@brad-decker Done [here](https://github.com/MetaMask/metamask-extension/pull/15549/commits/d1ae5af192efb5c81d7eefe7c028e6a1223c6e37)